locales/sv/stream.ftl

```diff
@@ -24,45 +45,140 @@ comment-count-text =
+comments-permalinkView-section =
+  .aria-label = Enskild konversation
 comments-permalinkView-viewFullDiscussion = Visa hela diskussionen
 comments-permalinkView-commentRemovedOrDoesNotExist = Den här kommentaren har tagits bort eller finns inte.
 
+comments-permalinkView-reportIllegalContent-title = Anmäl olagligt innehåll
+comments-permalinkView-reportIllegalContent-description = Vänligen fyll i detta formulär så gott det går så att vårt modereringsteam kan fatta ett beslut och vid behov konsultera vår webbplats juridiska avdelning.
+comments-permalinkView-reportIllegalContent-reportingComment = Du anmäler denna kommentar
+comments-permalinkView-reportIllegalContent-lawBrokenDescription-inputLabel = Vilken lag tror du har brutits? (obligatoriskt)
+comments-permalinkView-reportIllegalContent-additionalInformation-inputLabel = Inkludera ytterligare information om varför denna kommentar är olaglig (obligatoriskt)
+comments-permalinkView-reportIllegalContent-additionalInformation-helperText = Alla detaljer du inkluderar kommer att hjälpa oss att undersöka detta vidare
+comments-permalinkView-reportIllegalContent-additionalComments-inputLabel = Vill du anmäla några andra kommentarer för att innehålla olagligt innehåll?
+comments-permalinkView-reportIllegalContent-bonafideBelief-checkbox = Jag tror att informationen som ingår i denna rapport är korrekt och komplett
+comments-permalinkView-reportIllegalContent-additionalComments-addCommentURLButton = <Button></Button>Lägg till
+comments-permalinkView-reportIllegalContent-additionalComment-commentURLButton = Kommentarens URL
+comments-permalinkView-reportIllegalContent-additionalComments-deleteButton = <icon></icon> Ta bort
+comments-permalinkView-reportIllegalContent-submit = Skicka in rapport
+comments-permalinkView-reportIllegalContent-additionalComments-commentNotFoundError = Kommentaren hittades inte. Vänligen ange en giltig kommentars-URL
+comments-permalinkView-reportIllegalContent-additionalComments-validCommentURLError = Detta är inte en giltig URL. Vänligen ange en giltig kommentars-URL
+comments-permalinkView-reportIllegalContent-additionalComments-uniqueCommentURLError = Du har redan lagt till denna kommentar i rapporten. Vänligen lägg till en unik kommentars-URL
+omments-permalinkView-reportIllegalContent-additionalComments-validCommentURLLengthError = Längden på ytterligare kommentars-URL överskrider maximalt.
+comments-permalinkView-reportIllegalContent-additionalComments-previouslyReportedCommentError = Du har tidigare anmält denna kommentar för att innehålla olagligt innehåll. Du kan bara anmäla en kommentar av denna anledning en gång.
+comments-permalinkView-reportIllegalContent-confirmation-successHeader = Vi har mottagit din anmälan om olagligt innehåll
+comments-permalinkView-reportIllegalContent-confirmation-description = Your report will now be reviewed by our moderation team. You will receive a notification once a decision is made. If the content is 
```

Comment:
I noticed that this translation is still in English. Did you want to translate it?

Reply:
Thanks for the review @losowsky! 
You are right, that string was missed. I pushed additional commit to resolve the translation now.

Reply:
Thank you!